### DISCUSSION TITLE
Android SVG no Text

### DISCUSSION THREAD

**Discussion by AndreasSteibl** [AngusJohnson/Image32#59] [General]
Hello i have a quick question, i just tried the TImage32 with SVG (Img32.SVG.Reader) and on Windows64 (FMX) all works nice but the same example on Android doesn't show the text.
It is a very very simple SVG only with 
`<text id="value" x="58" y="38" width="125" >value</text>`

Is it a known problem?
regards
andreas

**Comment by AndreasSteibl** [comment]
Ok found my problem.
On non windows, there is no standard Font loaded into the FontManager.
Loaded one from a ressource, and now it works.

Do you know if i have to add every font into the ressource?
Are there no fonts on thze android device i can use?

**Comment by AngusJohnson** [comment]
> Ok found my problem.

👍 
And I'd just finished another app to demo FMX and SVG, and works just fine on my Android phone 😁...
[FMX_SVG_Text.zip](https://github.com/user-attachments/files/15884625/FMX_SVG_Text.zip)

> On non windows, there is no standard Font loaded into the FontManager.

Correct.

> Do you know if i have to add every font into the ressource?

No. Image32's FontManager will find the best/closest match from the available fonts, even if there's only one.

> Are there no fonts on thze android device i can use?

I'm not sure how to access system fonts on Android, though I assume it's possible. 
But ISTM the simplest/safest solution is to compile one or more freeware fonts into resources and use those.
